src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidator.java

```diff
@@ -160,10 +160,15 @@ private void validateTrustDetails(OverseasEntitySubmissionDto overseasEntitySubm
 
             if (!overseasEntitySubmissionDto.isForUpdateOrRemove()) {
                 // Note that this validation is only done for registrations
-                trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
+                trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext, false);
                 historicalBeneficialOwnerValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
                 trustCorporateValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext);
             }
+
+            else {
+                trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext, true);
```

Comment:
You can call the `trustIndividualValidator.validate() `method outside of the 'if' 'else' statement just passing in the `isForUpdateOrRemove` boolean value.
i.e.
```suggestion
                trustIndividualValidator.validate(overseasEntitySubmissionDto.getTrusts(), errors, loggingContext,  overseasEntitySubmissionDto.isForUpdateOrRemove());
```

Reply:
I've amended this now.